geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/geo/validators/KnownNameOrExpressionValidator.scala

```diff
@@ -0,0 +1,25 @@
+package org.geomesa.nifi.geo.validators
+
+import org.apache.nifi.components.{ValidationContext, ValidationResult, Validator}
+
+class KnownNameOrExpressionValidator(names: List[String]) extends Validator{
```

Comment:
this could use a little formatting cleanup:

- [ ] space before the `{` after `extends Validator`
- [ ] the `validate` method looks like it could be a single line, if not then indent like [this](https://github.com/locationtech/geomesa/blob/master/geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/ConverterInputFormat.scala#L91-L94)
- [ ] add `{}` around the validate function body, since it's not a single line
- [ ] add `{}` around your if/else clauses
- [ ] add new line at the end of the file